### PR TITLE
CU-8j90d4 base code

### PR DESCRIPTION
**Pull request by Bernix01**

### CODE REVIEW

**Comment by Bernix01** [comment]
Task linked: [CU-8j90d4 Inicializar y configurar ambiente Móvil Personal Seguridad](https://app.clickup.com/t/8j90d4)

**Comment by Bernix01** [comment]
@whugo43 GitHub actions files are missing? algo please move the app code to the root directory.

**Review by Bernix01 — COMMENTED** [review]
GitHub actions files are missing? algo please move the app code to the root directory.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (Bernix01) [inline .github/workflows/nodejs.yml]: please change this to allow the workflow to be executed on push to any branches, this workflow does ui testing and code style checks so it should not be bounded to an specific branch.

```yml

on: [push]

```

**Comment by Bernix01** [comment]
Merge conflicts must be resolved btw

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    # Runs a single command using the runners shell
+    - name: Run a one-line script
+      run: echo Hello, world!
+
+    # Runs a set of commands using the runners shell
+    - name: Run a multi-line script
+      run: |
+        echo Add other actions to build,
+        echo test, and deploy your project.
```
Comment (Bernix01) [inline .github/workflows/nodejs.yml]: There is a lint error, there must be a newline at the end of the file

Comment (Bernix01) [inline .github/workflows/nodejs.yml]: The script is just echoing some random text. Please update this so it executes the ui tests and lint checks

i.e:
```yml
      run: |
        npm test \
        npm run lint
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+
+TBD.
+
+## The stack
+
+This project is made with [ionic](https://ionicframework.com/) and based on the [With Graphql And React Hooks].
+As communnications are being done with [Graphql](https://graphql.org) this project uses [graphql-hooks](https://github.com/nearform/graphql-hooks) library to integrate graphql, react, react hooks with ease.
+
+Usefull links:
+
```
Comment (Bernix01) [inline README.md]: please add ionic docs link (related to the selected framework)

File: package.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.4.1",
+    "styled-components": "^5.1.1",
+    "typescript": "3.8.3"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```
Comment (Bernix01) [inline package.json]: please add the lint script so lint checks can be executed locally and by the CI

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (Bernix01) [reply .github/workflows/nodejs.yml]: done

**Review by Oswagui — APPROVED** [review]
Review:
- Clone without problems
- Test chek in teams meetting.
- General check in teams meeting.
- Recomendation to Readme conflict.

**Pull request merged**